src/main/java/pwr/asi/java/hermes/dtos/MailDTO.java

```diff
@@ -0,0 +1,80 @@
+package pwr.asi.java.hermes.dtos;
+
+import pwr.asi.java.hermes.entities.Mail;
+import pwr.asi.java.hermes.entities.Template;
+
+import java.time.LocalDateTime;
+import java.util.Objects;
+
+public class MailDTO {
+
+    private Template formattedTemplate;
+    public static int globalCounter = 0;
+    private int individualCounter;
```

Comment:
What are the points of these counters?
Also DTO should not contain any logic - it's for data transfer only